mleap-xgboost/src/main/scala/ml/combust/mleap/xgboost/bundle/ops/XGBoostRegressionOp.scala

```diff
@@ -0,0 +1,46 @@
+package ml.combust.mleap.xgboost.bundle.ops
+
+import java.io.ByteArrayInputStream
+import java.nio.file.Files
+
+import biz.k11i.xgboost.Predictor
+import ml.combust.bundle.BundleContext
+import ml.combust.bundle.dsl.{Model, Value}
+import ml.combust.bundle.op.OpModel
+import ml.combust.mleap.bundle.ops.MleapOp
+import ml.combust.mleap.runtime.MleapContext
+import ml.combust.mleap.xgboost.{XGBoostRegression, XGBoostRegressionModel}
+import resource._
+
+/**
+  * Created by hollinwilkins on 9/16/17.
+  */
+class XGBoostRegressionOp extends MleapOp[XGBoostRegression, XGBoostRegressionModel] {
+  override val Model: OpModel[MleapContext, XGBoostRegressionModel] = new OpModel[MleapContext, XGBoostRegressionModel] {
+    override val klazz: Class[XGBoostRegressionModel] = classOf[XGBoostRegressionModel]
+
+    override def opName: String = "xgboost.regression"
```

Comment:
should we include these in Bundle.BuiltinOps like all others for consistency?

Reply:
See above